src/test/unit/scheduler_unittest.cc

```diff
@@ -70,6 +70,7 @@ extern "C" {
     // set up micros() to simulate time
     uint32_t simulatedTime = 0;
     uint32_t micros(void) { return simulatedTime; }
+    uint32_t millis(void) { return simulatedTime/1000; }
```

Comment:
Please add comment that this implementation is good for short unittest only (it will behave terribly after 2^23us, someone may reuse it in flight code)